features/us65_delete_borrower_personal_info.feature

```diff
@@ -0,0 +1,19 @@
+@us65_delete_borrower_personal_info
```

Comment:
Would be tempted to split this tag up into two seperate tags like it's done in the other feature files, so have one with just the story number @us65 which is nice and quick and easy to use for those in the know. But also the longer one using the story name @delete_borrower_personal_info for those who might not be up to date with the user story number being used.